scripts/setup-env.js

```diff
@@ -0,0 +1,46 @@
+let fs = require('fs')
+let exec = require('child_process').exec;
+
+if (process.env.ISSUER === undefined || process.env.CLIENT_ID === undefined) {
+	console.log("[ERROR] Please set the ISSUER and CLIENT_ID Environment variables");
+	return;
+}
+
+updateConfig("custom-login");
+updateConfig("okta-hosted-login");
+cloneRepository("https://github.com/okta/samples-java-spring-mvc.git", "samples-java-spring-mvc");
+cloneRepository("https://github.com/okta/okta-oidc-tck.git", "okta-oidc-tck");
+
+function updateConfig(directory) {
+    let file = __dirname + "/../" + directory + "/src/app/.samples.config.ts"
```

Comment:
Use path.join for consistency across platforms https://nodejs.org/api/path.html#path_path_join_paths